lib/CleantalkSP/SpbctWP/Settings/DirExclusion.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace CleantalkSP\SpbctWP\Settings;
+
+use CleantalkSP\SpbctWP\Helpers\CSV;
+
+class DirExclusion
```

Comment:
Rename method and functions from dirExclusions to filesScanDirExclusions. This is not obvious, what the class used for without docs.